src/unwind.cc

```diff
@@ -59,32 +59,46 @@ void unwind_init_sample(UnwindState *us, const uint64_t *sample_regs,
 
 DDRes unwindstate_unwind(UnwindState *us) {
   DDRes res = ddres_init();
+  Process &process = us->process_hdr.get(us->pid);
+  bool avoid_new_attach = false;
+  if (us->process_hdr.process_count() > us->max_pids) {
```

Comment:
`process_count()` might not be equal to the number of attached dwfl objects.

Reply:
Yes. Agreed.

Reply:
I am not pushing this further as making the control perfect will take quite an extra bit of work. I understand the current limitations.